src/common/system_signal_handling.c

```diff
@@ -180,7 +191,47 @@ void dt_set_signal_handlers()
     fprintf(stderr, "[dt_set_signal_handlers] error: signal(SIGSEGV) returned SIG_ERR: %i (%s)\n", errsv,
             strerror(errsv));
   }
-#endif
+#else
+  /*
+  Set up exception handler for backtrace on Windows
+  works when there is NO SIGSEGV handler installed
+
+  Must be set up _after_: InitializeMagick(),
+  as GraphicsMagick is overwriting SetUnhandledExceptionFilter
+  an all other signals in its init function
+  The first invokation of the dt_set_signal_handlers() is _before_ InitializeMagick(),
+  therefore no point to set up exception handler
+  It should be set up only at the second invokation of  dt_set_signal_handlers(),
+  as at that time InitializeMagick() is done, and exception handler will work
+  Eventually InitializeMagick() should be fixed upstream not to ignore existing exception handlers
+  */
+  if(_times_handlers_were_set > 1)
+  {
+    ExcHndlInit();
+
+    gchar *name_used;
+    int fout;
+    bool ok;
+
+    if((fout = g_file_open_tmp("darktable_bt_XXXXXX.txt", &name_used, NULL)) == -1)
+      fout = STDOUT_FILENO; // just print everything to stdout
+
+    if(fout != STDOUT_FILENO) close(fout);
+
+    // Set up logfile name
+    ok = ExcHndlSetLogFileNameA(name_used);
+    if(!ok)
+    {
+      g_printerr("backtrace logfile cannot be set to %s\n", name_used);
```

Comment:
And if it succeeded, when saving backtrace, does it notify where did it save it to?